client/me/help/help-contact-closed/detail.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+export default localize( ( props ) => {
+	const {	icon, children } = props;
+
+	return (
+		<div className="help-contact-closed__detail">
+			<div>
```

Comment:
You could try `flex-shrink: 0` on the gridicon. It's a minor detail, but generally reducing the markup to the more semantic minimum is good.